src/controllers/_base.js

```diff
@@ -83,7 +89,7 @@ const renderPage = (request, response, options) => {
   // Handle un-session-ed accesses to '/success' a little differently. The
   // user may have bookmarked this page, thinking they could see their
   // registration code again. Give them an error page that says otherwise.
-  if (options.path === 'success') {
+  if (options.path === 'submitted-renewal-success') {
```

Comment:
Changing this means the `success` page will no longer return 403 if visited when un-session-ed, and instead gives this:
![undefined_trap](https://github.com/Scottish-Natural-Heritage/trap-registration/assets/44615789/a6a62c88-46d1-4521-8ed0-9e8893523c6f)